README.md

```diff
@@ -1 +1,34 @@
-# Portfolio_module
\ No newline at end of file
+Portfolio
+In this project I design the mobile version of the portfolio.
+It has two section one is toolbar which is navbar and the other is heading section. 
```

Comment:
- [ ]OPTIONAL: Please consider updating your readme file adding more information, headings, properly aligning sections. Currently, readme looks very confusing:

Your README  | An example of a professional README
------------- | -------------
<img width="890" alt="Screenshot 2022-03-18 at 15 15 47" src="https://user-images.githubusercontent.com/77038610/158993574-a2b79634-db7a-4e7a-8900-3c656be97a78.png">  | <img width="609" alt="Screenshot 2022-03-17 at 20 45 31" src="https://user-images.githubusercontent.com/77038610/158851176-0a84e776-787b-439d-90ee-13e918fb20fb.png">

[Here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) you can read more and learn about formatting `.md` documents on Github